packages/airnode-examples/integrations/coinmarketcap/README.md

```diff
@@ -0,0 +1,15 @@
+# CoinMarketCap example integration
```

Comment:
```suggestion
# Authenticated CoinMarketCap example integration
```

or put "authenticated" somewhere (maybe even folder name `authenticated-coinmarketcap`) so it's easier to find for users.

Reply:
👍 title updated and folders renamed